src/vs/workbench/browser/parts/views/viewPane.ts

```diff
@@ -655,6 +656,10 @@ export abstract class FilterViewPane extends ViewPane {
 	private dimension: Dimension | undefined;
 	private filterContainer: HTMLElement | undefined;
 
+	private readonly _onDispose = this._register(new Emitter<void>());
+	protected readonly _onDidBlurResults = this._register(new EventMultiplexer<void>());
```

Comment:
I'm not a fan of this pattern, the subclass has to "just know" that they need to hook into this

Reply:
I think you should either change this so that the subclass does the registration and handles all the work, or, force the subclass to set this up properly. Also, I'm wondering whether we would have a FilterViewPane that doesn't quite fit this pattern and doesn't want to use this, but I can't think of a case that makes sense right now.

Reply:
Fix in commit https://github.com/microsoft/vscode/pull/187077/commits/a24cc1ba3ccf5deb0f07942386ba5dc7357294fe